app/controllers/internal/chat_channels_controller.rb

```diff
@@ -23,12 +23,22 @@ def update
       redirect_back(fallback_location: "/internal/chat_channels")
     end
 
+    def remove_user
+      @chat_channel = ChatChannel.find(params[:id])
+      @chat_channel.remove_user(user_by_param)
+      redirect_back(fallback_location: "/internal/chat_channels")
+    end
+
     private
 
     def users_by_param
       User.where(username: chat_channel_params[:usernames_string].downcase.delete(" ").split(","))
     end
 
+    def user_by_param
+      User.where(username: chat_channel_params[:usernames_string])
```

Comment:
This should be `find_by` so we only pass in a singular User record.

```suggestion
      User.find_by(username: chat_channel_params[:username_string])
```